samples/app-anonymous-users/csharp/TeamsAppManifest/manifest.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.teams.sample.appAnonymousUsers",
+  "developer": {
+    "name": "appAnonymousUsers",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "name": {
+    "short": "Anonymous User Support",
+    "full": "Anonymous User Support"
+  },
+  "description": {
+    "short": "This sample shows Anonymous Users Support In Meeting.",
+    "full": "This sample shows Anonymous Users Support In Meeting."
```

Comment:
This sample shows anonymous users support in meeting apps.

Reply:
Fixed